diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocComments.kt

```diff
@@ -80,13 +82,27 @@ class KdocComments(private val configRules: List<RulesConfig>) : Rule("kdoc-comm
                 FILE -> checkTopLevelDoc(node)
                 CLASS -> checkClassElements(node)
                 VALUE_PARAMETER -> checkValueParameter(node)
+                VALUE_PARAMETER_LIST -> checkParameterList(node)
                 else -> {
                     // this is a generated else block
                 }
             }
         }
     }
 
+    private fun checkParameterList(node: ASTNode) {
+        val kdocBeforeClass = node.parent({ it.elementType == CLASS })?.findChildByType(KDOC) ?: return
+        val propertiesInKdoc = kdocBeforeClass.kDocTags()?.filter { it.knownTag == KDocKnownTag.PROPERTY}?.mapNotNull {it.getSubjectName()}
+        val propertyNames = (node.psi as KtParameterList).parameters.filter {
+            it.node.getFirstChildWithType(MODIFIER_LIST).isAccessibleOutside() }
+                .mapNotNull { it.nameIdentifier?.text }
+        propertiesInKdoc?.let { kdocProperties ->
```

Comment:
Maybe just `filterNot { it in propertyNames }` instead of collections subtraction?